docs/messaging/messaging-spans.md

```diff
@@ -348,23 +342,13 @@ size should be used.
 
 **[13]:** If a custom value is used, it MUST be of low cardinality.
 
-**[14]:** The value SHOULD be normalized to lowercase.
-
-**[15]:** Only for messaging systems and frameworks that support more than one protocol.
-
-**[16]:** `network.protocol.version` refers to the version of the protocol used and might be different from the protocol client's version. If the HTTP client has a version of `0.27.2`, but sends HTTP version `1.1`, this attribute should be set to `1.1`.
-
-**[17]:** The value SHOULD be normalized to lowercase.
+**[14]:** Network peer address and port are important when the application interacts with individual intermediary nodes directly, for example, when messaging intermediary is self-hosted. Fully managed messaging systems that don't expose information about individual nodes SHOULD NOT set this attribute. Semantic conventions for individual messaging systems SHOULD document whether `network.peer.*` attributes should be captured. If a messaging operation involved multiple network calls (for example retries), the address of the last contacted node SHOULD be used.
```

Comment:
Can client instrumentation know whether it's interacting with a self-hosted or fully-managed?

If for a given system possibly both self-hosted and fully-managed offerings exist, what should client instrumentation do? As in may cases it can likely not know whether a given address refers to a self-hosted or managed instance.

Reply:
The intention is to let individual systems decide if/how/when `network.peer.*` should be collected. I rephased it a bit to make it more clear and update this sentence to "Instrumentations for messaging systems that don't provide self-hosted mode SHOULD NOT set this attribute." LMK what you think.

Reply:
@pyohannes could you please take a look and lmk if you have any objections?

Reply:
This looks good.